app/components/Cards/PricingCard.tsx

```diff
@@ -0,0 +1,31 @@
+"use client";
+import classes from "./PricingCard.module.css";
+import checkpoint from "../../../public/Icons/checkpoint_icon.svg";
+import Image from "next/image";
+
+const PricingCard = (props: any) => {
```

Comment:
Maybe its better to use destructuring for props. For example const PricingCard = ({ typePlan }) => {